project2/styles.css

```diff
@@ -0,0 +1,134 @@
+* {
+    box-sizing: border-box;
+    /* border: 3px solid red; */
+}
+
+h2 {}
+
+.u-button-reset {
+    background: rgba(77, 28, 255, 0.527);
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+h1 {
+    font-family: 'Righteous', cursive;
+    font-size: 80px;
+    text-align: center;
+    margin: 50px 0;
+}
+
+h2 {
+    font-family: 'Overpass', sans-serif;
+    font-size: 20px;
+}
+
+nav {
+    margin-bottom: 60px;
+    font-family: 'Overpass', sans-serif;
+}
+
+ul {
+    list-style: none;
+    text-align: center;
+}
+
+li+li {
+    margin-left: 100px;
+}
+
+li {
+    display: inline-block;
+}
+
+li a {
+    color: whitesmoke;
+    margin-right: 4px;
+    margin-left: 4px;
+    margin-bottom: 4px;
+    padding: 5px;
+    text-decoration: none;
+}
+
+li a:hover,
+li a:focus {
+    color: hsl(194, 100%, 50%);
+}
+
+body {
```

Comment:
The `body` rule should be at the top of your stylesheet after the `box-sizing` rule. You want your most general, generic rules at the top and to get more specific as you go deeper into the stylesheet.